SETUP.md

```diff
@@ -2,42 +2,14 @@
 
 This guide is to get you setup with the tools you'll need for development.
 
-**Date**: May 24, 2002
-
-**Platform**: Fedora 35, 64-bit
```

Comment:
I'd recommend just leaving the install instructions here and making a separate one for mac. Generally there's a setup guide per major distribution in most repos, and there's no reason to remove a previously tested workflow here.

Reply:
@marcinwadon wants to avoid that, preferring to generalize the instructions.

Reply:
Re-added OS specific links and instructions in separate sections.